tests/python/unittest/test_meta_schedule_integration.py

```diff
@@ -149,5 +151,49 @@ def extract_task_qbert():
         assert "vnni" in annotations["schedule_rule"]
 
 
+def extract_task_arm_conv2d_nchwc():
```

Comment:
Are we sure this runs @masahi? There's no `test_` prefix

Reply:
oops thanks for pointing this out, I added a fix in https://github.com/apache/tvm/pull/10773